test/test-util.ts

```diff
@@ -203,6 +204,23 @@ export function createRawNftContractBaseNft(
   };
 }
 
+export function verifyNftContractMetadata(
+  actualNftContract: NftContract,
+  expectedNftContract: NftContract,
+  address: string,
```

Comment:
why do we need all the additional variables in the params here?

i'm thinking that we would just do
`expect(actualNftContract.address).toEqual(expectedNftContract.address);`

Reply:
Playing with the API I saw that every element in the `searchContractMetadata` endpoint response has the same structure as the response from the `getNftContractMetadata` endpoint which was unit tested using the following function 

![image](https://user-images.githubusercontent.com/42520800/199609066-c94a1990-e105-4321-a7b4-0b6f29b9b211.png)
![image](https://user-images.githubusercontent.com/42520800/199609728-c378c56f-b1c2-4b0c-9d10-275a6642ff7b.png)

So to keep the testing method consistent I decided to reuse the `verifyNftContractMetadata` function for the `searchContractMetadata` tests too.

Reply:
ah I see what you mean, thanks for clarifying!

please leave this as is, i'll refactor this later!

Reply:
Ok, nice! Thank you!